project/BuildDependencies/scripts/0_package.target-win10-arm.list

```diff
@@ -1,37 +1,39 @@
 ; package.list should contain all file names of packages which are already in the "new" package format (the package contains the paths to the libraries
 ; and headers relative to the XBMC root directory, stored in a zip container)
-; example sqlite-3.7.12.1-win32.7z:
+; example sqlite-3.7.12.1-arm.7z:
 ; -> (package name)\(relative path to file)
-; -> sqlite-3.7.12.1-win32\project\BuildDependencies\lib\sqlite3.lib
-; -> sqlite-3.7.12.1-win32\system\sqlite3.dll
+; -> sqlite-3.7.12.1-arm\project\BuildDependencies\lib\sqlite3.lib
+; -> sqlite-3.7.12.1-arm\system\sqlite3.dll
 ; -> ...
 ;PLEASE KEEP THIS LIST IN ALPHABETICAL ORDER!
-crossguid-fef89a4-win10-ARM-v140.7z
-curl-7.59.0-win10-ARM-v141.7z
-flatbuffers-1.9.0.7z
-fmt-3.0.1-win10-ARM-v140.7z
-freetype-2.8-win10-ARM-v140.7z
-fstrcmp-0.7-win10-ARM-v141.7z
-lcms2-2.8-win10-ARM-v140.7z
-libaacs-0.9.0-win10-ARM-v141.7z
-libass-0.13.6-win10-ARM-v140.7z
-libbluray-1.0.2-win10-ARM-v141.7z
-libcdio-0.94-win10-ARM-v140.7z
-libfribidi-0.19.7-win10-ARM-v140.7z
-libiconv-1.15-win10-ARM-v141.7z
-libmicrohttpd-0.9.55-win10-ARM-v140.7z
-libnfs-3.0.0-win10-ARM-v141.7z
-libplist-1.12-win10-ARM-v140.7z
-libxml2-2.9.4-win10-ARM-v141.7z
-libxslt-1.1.29-win10-ARM-v140.7z
-lzo2-2.09-win10-ARM-v140.7z
-mini_wdk-10.0.14393.0-win32.7z
-mysql-connector-c-6.1.9-win10-ARM-v141.7z
-openssl-1.0.2o-win10-ARM-v141.7z
-pcre-8.40-win10-ARM-v141.7z
-rapidjson-1.1.0-win32.7z
-shairplay-ce80e00-win10-ARM-v140.7z
-sqlite-3170000-win10-ARM-v140.7z
-taglib-1.11.1-win10-ARM-v140.7z
-tinyxml-2.6.2-win10-ARM-v140.7z
-zlib-1.2.11-win10-ARM-v140.7z
+crossguid-2.0.3-win10-arm-v141-20191222.7z
+curl-7.67.0-win10-arm-v141-20191222.7z
+flatbuffers-1.11.0-20191222.7z
+fmt-6.1.2-win10-arm-v141-20191222.7z
+freetype-2.10.1-win10-arm-v141-20191222.7z
+fstrcmp-0.7-win10-arm-v141-20191222.7z
+lcms2-2.9-win10-arm-v141-20191222.7z
+libaacs-0.9.0-win10-arm-v141-20191222.7z
+libass-0.14.0-win10-arm-v141-20191222.7z
+libbdplus-0.1.2-win10-arm-v141-20191222.7z
```

Comment:
Should this be bundled with Kodi?

Reply:
I don't see why not, it's used by libbluray so we need it

Reply:
No libbluray works fine without is. Also it only dynamically loads this library so one could place it at the correct location and it will work too, except for WindowsStore I know.
I'm just not sure if it's a good idea to bundle it.